lib/db/schema/patch-092-093.sql

```diff
@@ -0,0 +1,269 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('92');
+
+CREATE TABLE IF NOT EXISTS `clientInstancesExtraData` (
```

Comment:
@shane-tomlinson from our recent conversation, it sounded like you'd prefer to try to add new columns to the existing table rather than go down this "second table with extra fields in it" route; is that accurate?

Reply:
> it sounded like you'd prefer to try to add new columns to the existing table rather than go down this "second table with extra fields in it" route.

That I would, but doing so depends on having a DB migration path to doing so!